### ISSUE TITLE
Optionalize ddMicrobunches

### ISSUE THREAD

**Issue by balerion** [momentoscope/HextofOfflineAnalyzer#19]
I have been thinking of making the whole ddMicrobunches dataframe optional. Is this even necessary at all? we should have all the data in the dd dataframe anyway. If needed for debugging purposes, we can generate it optionally. Or, we could always generate it but have it empty if not required?

Just a proposition, no need to fuss about this if not necessary.

**Comment by steinnymir** [comment]
I totally agree with this. The generation of ddMicrobunch is time and space consuming, while almost never used at all.

**Comment by balerion** [comment]
The ddMicrobunches is only used for diagnostics and for generating the histograms. We are looking into generating the histograms in other ways, using the dd dataframe, so this will be optionalized.
